app/scripts/controllers/emissionCtrl.js

```diff
@@ -146,8 +140,7 @@ var emissionCtrl = function($scope, $sce, walletService, libreService, $rootScop
         }
     }, 1000);
```

Comment:
Maybe move to const

Reply:
1000 means one second for timer (decreasing values each second)
it shouldn't be changed, so I think we can leave it as is